linked_list/linked_list.py

```diff
@@ -0,0 +1,49 @@
+from node import Node
+
+class LinkedList(object):
+    """Creates a data structure to link data together by attributes"""
+    def __init__(self, iter=[]):
+        self._current = None
+        self.head = None
+        self._len = 0
+
+        for item in reversed(iter):
+            self.insert(item)
+
+    def __repr__(self):
+        """Prints a the head pointer of the current Linked-List"""
+        return 'Head of Node: {}'.format(self.head.val)
+
+    def __len__(self):
+        """Returns LinkedLists Length"""
+        return self._len
+
+    def insert(self, val):
+        """Inserts new Node into Linked-List and sets it as a head pointer"""
+        self.head = Node(val, self.head)
+        self._len += 1
+
+    def find(self, value):
+        """Finds users input within Linked-List"""
+        self._current = self.head
```

Comment:
Since this `_current` property is not being used in other methods, you could use a normal variable instead.